main/src/com/google/refine/commands/browsing/GetScatterplotCommand.java

```diff
@@ -145,7 +145,7 @@ public void setDimY(String dim) {
         @JsonProperty(ScatterplotFacet.ROTATION)
         public void setRotation(Object rotation) {
         	try {
-        		this.rotation = Integer.parseInt(rotation.toString());
+        		this.rotation = ScatterplotFacet.ScatterplotFacetConfig.getRotation(rotation.toString());
         	} catch(NumberFormatException e) {
```

Comment:
It looks like this NumberFormatException can no longer be thrown, so perhaps we could remove this `try`/`catch` block?

Reply:
Sure. Then the process will be handled by getRotation().